packages/react-components/react-motion/src/utils/animateAtoms.ts

```diff
@@ -38,6 +38,18 @@ export function animateAtoms(
       });
     },
     set onfinish(callback: () => void) {
+      // Heads up!
+      // Jest uses jsdom as the default environment, which doesn't support the Web Animations API. This no-op is
+      // necessary to avoid errors in tests.
+      //
+      // See https://github.com/microsoft/fluentui/issues/31593
```

Comment:
is there an issue in JSDOM for this too?

Reply:
yes

Reply:
```suggestion
      // See https://github.com/microsoft/fluentui/issues/31593
      // See https://github.com/jsdom/jsdom/issues/3429
```